modules/network-observability-flowcollector-api-specifications.adoc

```diff
@@ -2538,7 +2539,7 @@ configuration, you can disable it and install your own instead.
 | `boolean`
 | Set `enable` to `true` to deploy network policies on the namespaces used by Network Observability (main and privileged). It is disabled by default.
 These network policies better isolate the Network Observability components to prevent undesired connections to them.
-Either enable it, or create your own network policy for Network Observability.
+We recommend you either enable it, or create your own network policy for Network Observability.
```

Comment:
I've seen caution about using "recommend" language in docs. Can this be stated in a way that avoids it?

Reply:
I think we can maybe write instead: To increase the security of connections, enable this option or create your own network policy.

I've added this in a comment in @jotak 's PR https://github.com/netobserv/network-observability-operator/pull/827#pullrequestreview-2428121065, along with your other feedbacks, so we can make sure the upstream/downstream are synchronized. Thanks for your review Jeana!